### ISSUE TITLE
Locked slots reset on relog in multiplayer.

### ISSUE THREAD

**Issue by AngelOfCreation** [blackd/Inventory-Profiles#453]
### Have you asked for help on discord?

YES

### Minecraft Version/s

1.20.1

### Modloader

Forge

### Modloader Version

47.3.10

### Operating System

Windows

### List of all other installed mods

https://pastebin.com/MkhBEjzM

### Describe the problem

When relogging on a server, locked slots reset and vanish. It's not just visual, items in those slots get sorted when you press the sort buttons. Upon changing dimensions the locked slots return but only until you relog again.

### Steps to reproduce

Join a server
Apply locked slots
Exit server
Rejoin server
Locked slots are gone.

### I have tried to reproduce the problem with only or without "Inventory Profiles Next" and its dependencies installed.

- [ ] I have tested without 'Inventory Profiles Next' and its dependencies
- [ ] I have tested with only 'Inventory Profiles Next' and its dependencies and no other mods

**Comment by blackd** [comment]
yeah with that quantity of mods please do test with no other mods

**Comment by IIIIIVIVIX** [comment]
Is this solved yet? I'm getting the same problems...

**Comment by blackd** [comment]
after login press r+y this will reload the configuration and "fix" it.
